--- source_subfolder/src/core/runner_coroutine.c	2018-06-07 15:28:12.675875819 +0200
+++ source_subfolder/src/core/runner_coroutine.c.patched	2018-06-07 15:30:50.951337898 +0200
@@ -297,7 +297,7 @@
     if (rc < 0)
         cr_panic("Could not initialize test context: %s", strerror(-rc));
 
-    struct bxf_spawn_params sp = {
+    struct bxf_spawn_params_s sp = {
         .fn              = run_test_child,
         .callback        = death_callback,
         .inherit.context = inst_ctx,
