--- source_subfolder/CMakeLists.txt	2018-06-07 10:32:56.143553379 +0200
+++ source_subfolder/CMakeLists.txt.patched	2018-06-07 12:40:12.039831322 +0200
@@ -18,7 +18,6 @@
 # Initialization
 
 include (Options)
-include (Submodules)
 include (Capabilities)
 include (Subprojects)
 include (PackageUtils)
--- source_subfolder/src/CMakeLists.txt	2017-04-08 14:22:26.000000000 +0200
+++ source_subfolder/src/CMakeLists.txt.patched	2018-06-06 17:01:11.145434126 +0200
@@ -75,13 +75,6 @@
   src/string/extglobmatch.h
   src/string/brz.c
   src/string/brz.h
-
-  dependencies/nanopb/pb_common.c
-  dependencies/nanopb/pb_common.h
-  dependencies/nanopb/pb_encode.c
-  dependencies/nanopb/pb_encode.h
-  dependencies/nanopb/pb_decode.c
-  dependencies/nanopb/pb_decode.h
 )
 
 if (THEORIES)
