--- source_subfolder/src/CMakeLists.txt	2017-04-08 14:22:26.000000000 +0200
+++ source_subfolder/src/CMakeLists.txt.patched	2018-06-06 17:01:11.145434126 +0200
@@ -75,13 +75,6 @@
   src/string/extglobmatch.h
   src/string/brz.c
   src/string/brz.h
-
-  dependencies/nanopb/pb_common.c
-  dependencies/nanopb/pb_common.h
-  dependencies/nanopb/pb_encode.c
-  dependencies/nanopb/pb_encode.h
-  dependencies/nanopb/pb_decode.c
-  dependencies/nanopb/pb_decode.h
 )
 
 if (THEORIES)
--- source_subfolder/CMakeLists.txt	2018-06-07 17:31:49.818768109 +0200
+++ source_subfolder/CMakeLists.txt.patched	2018-06-07 17:32:58.802426241 +0200
@@ -18,7 +18,6 @@
 # Initialization
 
 include (Options)
-include (Submodules)
 include (Capabilities)
 include (Subprojects)
 include (PackageUtils)
@@ -155,6 +154,10 @@
   include_directories("${BOXFORT_INCLUDE_DIRS}")
   cr_link_libraries(criterion "${BOXFORT_LIBRARIES}" IF BOXFORT_FOUND)
 endif ()
+if (NANOPB_FOUND)
+  include_directories("${NANOPB_INCLUDE_DIRS}")
+  cr_link_libraries(criterion "${NANOPB_LIBRARIES}" IF NANOPB_FOUND)
+endif ()
 
 cr_link_libraries(criterion pthread IF NOT WIN32)
 cr_link_libraries(criterion rt IF HAVE_LIBRT)
